### ISSUE TITLE
Comentarios

### ISSUE THREAD

**Issue by Marc07** [Emer17/mips#3]
vamos escrever os comentarios apenas neste aqui, dai fica mais facil de visualizar

**Comment by Emerson-git** [comment]
pode ser entao.   ula_control não precisa de clock?

**Comment by Marc07** [comment]
Da pra por, mas não sei se muda muito.

**Comment by Emerson-git** [comment]
entao vou deixar sem mesmo.

**Comment by Emerson-git** [comment]
putz.. tem que fazer o Bo...vai ficar grande..kk

**Comment by Marc07** [comment]
vai to passando ele pra jpeg com os componentes e suas portas, to quase acabando dai eu posto ela para ver melhor as entradas e saidas.

**Comment by Emerson-git** [comment]
isso... pq eu to perdido aqui..

**Comment by Marc07** [comment]
Da uma olhada na imagem, qualqer coisa e so fala.

**Comment by Emerson-git** [comment]
nao entendi essa parte do brach com o zero.

**Comment by Marc07** [comment]
Toma cuidado com o rising_edge porque você esta esquecendo de colocar o parênteses.

**Comment by Marc07** [comment]
olha no bc, o branch so fica um se for uma instrução beq, se a saida zero da ula for 1 então o proximo valor do pc vem da soma de pc+4 com o valor do shifter.

**Comment by Emerson-git** [comment]
acho que vou pra uffs amanha de manha. estou com um pouco de dificuldade nos port maps do bo. vou precisar de ajuda...

**Comment by Marc07** [comment]
beleza, eu to fazendo ele também dai a gente vê amanhã bem certo.

**Comment by Emerson-git** [comment]
aham. voce viu que eu criei o bo ali neh.... só nao fiz muita coisa..kkk

**Comment by Marc07** [comment]
vi, eu também não fiz muito aqui.

**Comment by Emerson-git** [comment]
Quando você ver.. responda..

**Comment by Marc07** [comment]
Cheguei.

**Comment by Emerson-git** [comment]
Falei com o jacson pelo moodle.... ele disse que depois das 10 estaria por ai ...

**Comment by Emerson-git** [comment]
voce pode falar com ele se quiser, dependendo do que ele falar a gente arruma o que precisar e eu vou pra uffs uma noite dessas pra conversar com ele tambem..

**Comment by Marc07** [comment]
beleza, eu falo com ele.

**Comment by Emerson-git** [comment]
eu vou ficar por aqui. veja com ele o que a gente pode fazer pra testar... depois me responda.

**Comment by Marc07** [comment]
ele disse em qual sala ia estar

**Comment by Emerson-git** [comment]
nao.. quando pedi isso ele tinha saido do moodle.. acho que no bloco dos professores mesmo... nao tem aula neh?

**Comment by Marc07** [comment]
nao, eu vou la ver.

**Comment by Emerson-git** [comment]
blz

**Comment by Marc07** [comment]
Ele sugeriu que nos testassemos o mips completo, pelo que ele disse parece estar certo, embora eu ache que pelo jeito que ele viu a maioria nao fez com BO e BC, acho que fizeram direto no mips, ele disse que so ia testar as saidas da Ula para ver se estava correta.

**Comment by Emerson-git** [comment]
mas ai voce falou pra ele que a gente tentou e excedia a licença de estudante?

**Comment by Marc07** [comment]
nao nem lembrei, eu vou tentar testar por partes a tarde dai a gente ve se testa tudo.

**Comment by Emerson-git** [comment]
ta.. desculpe nao ter ido hoje... ontem eu sai pq tava mal mas hoje o celular nao despertou mesmo.... e eu achei q o jacson nao ia estar ai.... mas faz isso.. testa e me avisa... ai amanha de noite EU vou la e falo com ele. Quero ver sobre a resposta dos exercicios da prova tambem.

**Comment by Marc07** [comment]
é ele pediu se eu tinha feito dai eu disse que nos tinhamos respondido algumas sexta, dai ele disse que a prova ia ser encima delas e teria um pouco de vhdl

**Comment by Emerson-git** [comment]
exato... eu quero passar bem nessa materia.... pelo menos um 7..kk   amanha de noite eu vou pra uffs e vejo disso.... quinta a noite tem a rec de BD. vai fazer?

**Comment by Marc07** [comment]
Não, vou estuda pra ultima, hoje ele passou o calculo relacional eu nao achei dificil é mais facil que algebra.

**Comment by Emerson-git** [comment]
é que vc tirou 6 neh.. kk

**Comment by Marc07** [comment]
pois é, mas nesse eu acho que eu vou fazer uma cola pra tirar pelo menos um 7.

**Comment by Emerson-git** [comment]
certo.. eu pedi pq como eu vou a noite falar com ele, nao vai ter como conversar com voce. nao sei se vou pra uffs na quinta... acho melhor ir na sexta de manha.

**Comment by Marc07** [comment]
pode ser, qualquer coisa posta ai dai eu vejo quinta.

**Comment by Emerson-git** [comment]
certo. entao ficamos assim: voce testa o que puder e me manda um feedback por aqui mesmo... e amanha a noite eu vou pra la. pergunto o que precisar, vejo o que posso fazer, vejo da prova e, te respondo por aqui. Se puder levar o livro sexta... eu agradeço :+1:

**Comment by Emer17** [comment]
:+1:

**Comment by Marc07** [comment]
falo entao a gente se fala até sexta e boa prova quinta.

**Comment by Emerson-git** [comment]
Vlw. Até :facepunch:

**Comment by Emerson-git** [comment]
quando voce ver isso me responda... preciso saber se tem ou nao energia eletrica ai na uffs...

**Comment by Marc07** [comment]
Tem.
